### PR TITLE
Feature/gce templates

### PR DESCRIPTION
**Pull request by dereklwood**
default GCE templates

depends on #474

### CODE REVIEW

File: config/defaults/providers/google.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "google",
+  "description": "Google Compute Engine",
+  "providertype": "google",
+  "provisioner": {
+    "client_email": "",
```
Comment (albertshau) [inline config/defaults/providers/google.json]: we can remove the empty fields

**Comment by albertshau** [comment]
small comment, otherwise lgtm

File: config/defaults/plugins/providertypes/google.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        }
+      },
+      "required": [
+        [
+          "project_id",
+          "client_email",
+          "key_location",
+          "zone_name",
+          "ssh_keyfile",
+          "ssh_username"
```
Comment (wolf31o2) [inline config/defaults/plugins/providertypes/google.json]: This is required here, but not in the actual plugin. Is this on purpose?

File: config/defaults/plugins/providertypes/google.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        ]
+      ]
+    },
+    "user": {
+      "fields": {
+        "data_disk_size_gb": {
+          "default": "30",
+          "label": "Data disk size in Gb",
```
Comment (wolf31o2) [inline config/defaults/plugins/providertypes/google.json]: GB

File: config/defaults/providers/google.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    "key_location": "/opt/loom/provisioner/worker/plugins/providers/fog_provider/google-project-id.p12",
```
Comment (wolf31o2) [inline config/defaults/providers/google.json:6]: I have been using `/etc/loom/conf/credentials/filename.ext` as the new "default" location.

File: config/defaults/plugins/providertypes/google.json (hunk 2 above)
Comment (dereklwood) [inline config/defaults/plugins/providertypes/google.json]: good catch

File: config/defaults/providers/google.json (hunk 4 above)
Comment (dereklwood) [inline config/defaults/providers/google.json:6]: thats in our internal loom... but here i kept it `/opt/...` to be consistent with the others.  we can change them all in a subsequent PR

**Comment by dereklwood** [comment]
OTS

**Pull request merged**